README.md

```diff
@@ -1,8 +1,41 @@
-# iexec-blockchain-client
-A blockchain adapter API for iExec Smart-Contracts and other Ethereum things
+# iexec-blockchain-adapter-api
+
+## Overview
+
+A blockchain adapter API for iExec Smart-Contracts and other Ethereum things.
+
+The iExec Blockchain Adapter API is available as an OCI image on [Docker Hub](https://hub.docker.com/r/iexechub/iexec-blockchain-adapter-api/tags).
+
+To run properly, the iExec Blockchain Adapter API requires:
+* A reachable blockchain node URL hosting iExec smart contracts.
+* A reachable MongoDB instance to persist its data.
+* A reachable broker to match iExec orders.
+* An Ethereum wallet to interact with Smart Contracts on the blockchain network.
+
+## Configuration
+
+| Environment variables | Description | Values | Default value |
```

Comment:
```suggestion
| Environment variable | Description | Type | Default value |
```

Reply:
Thanks, I missed those updates in the last round